source/dub/project.d

```diff
@@ -108,22 +108,24 @@ class Project {
 		import dub.version_;
 		import dub.internal.dyaml.stdsumtype;
 
-		auto selected = mgr.readSelections(pack);
-		// Parsing error that will be displayed to the user or just no selections
-		if (selected.isNull())
+		auto lookupResult = mgr.readSelections(pack.path);
```

Comment:
How do we know here that the path is absolute ?
Loops like we should just change the signature of the function ?